csharp/tools/tracing-config.lua

```diff
@@ -84,6 +130,10 @@ function RegisterExtractorPack(id)
                 dotnetRunNeedsSeparator = false
                 dotnetRunInjectionIndex = i
             end
+            -- quote arguments if needed
```

Comment:
I think it would be best if we move this to the place where we build a command line argument string.  https://github.com/github/codeql/blob/e6bf8fa98a49aa8fe9a2318f1cb0b2467a877242/csharp/tools/tracing-config.lua#L163